src/main/webapp/intrigue-api/metacards/metacards.js

```diff
@@ -123,9 +123,9 @@ const typeDefs = `
     createMetacard(attrs: MetacardAttributesInput!): MetacardAttributes
     saveMetacard(id: ID!, attributes: MetacardAttributesInput!): MetacardAttributes
 
-    # TBD: Should only be used when...
+    # TBD: Should only be used when updating assocations with IDs
```

Comment:
```suggestion
    # Should only be used when updating associations with IDs or other expanded fields (i.e. lists, queries) where you need to set raw values but the schema would otherwise not allow you.
```